lib/mission_control/jobs/console/commands.rb

```diff
@@ -0,0 +1,38 @@
+require "irb/command"
+
+module MissionControl::Jobs
+  class ConnectTo < IRB::Command::Base
+    category "Mission control jobs"
+    description "Connect to a job server"
```

Comment:
You can also add `help_message` to commands to display more detailed messages when `help <cmd>` is called.

Reply:
Nice!!